packages/components/src/icon/index.js

```diff
@@ -8,24 +8,24 @@ import { cloneElement, createElement, Component, isValidElement } from '@wordpre
  */
 import { Dashicon, SVG } from '../';
 
-function Icon( { icon = null, size, className } ) {
+function Icon( { icon = null, size, className, ...additionalProps } ) {
```

Comment:
Technically `className` is redundant, since it would be applied now with the behavior of `additionalProps` anyways.